packages/modules/create-imodel/src/components/upload-image/UploadImage.tsx

```diff
@@ -86,7 +91,7 @@ export function UploadImage({
       ImageHelper.getOrientation(file, (orientation, fileBytes) => {
         setRotation(ImageHelper.convertRotationToDegrees(orientation));
         setImageUrl(reader.result as string);
-        onChange(fileBytes, file.type);
+        context?.onImageChange(fileBytes, file.type);
```

Comment:
forgot to use `onChange`

Reply:
this was onImageChange, not separate onChange

Reply:
what? No, the API had `onChange` prop so I think you should keep it? And user could subscribe on it too, if needed.